### PR TITLE
OSDOCS-11540: Registry - HCP Book Content Port

### PR DESCRIPTION
**Pull request by fmcdonal**
Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-11540

Server build here:
https://file.corp.redhat.com/fmcdonal/OSDOCS-11540/registry/index.html

Link to docs preview:
https://81266--ocpdocs-pr.netlify.app/openshift-dedicated/latest/registry/configuring-registry-operator.html
https://81266--ocpdocs-pr.netlify.app/openshift-dedicated/latest/registry/index.html
https://81266--ocpdocs-pr.netlify.app/openshift-enterprise/latest/registry/configuring-registry-operator.html
https://81266--ocpdocs-pr.netlify.app/openshift-enterprise/latest/registry/index.html
https://81266--ocpdocs-pr.netlify.app/openshift-rosa/latest/registry/configuring-registry-operator.html
https://81266--ocpdocs-pr.netlify.app/openshift-rosa/latest/registry/index.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by fmcdonal** [comment]
@arendej appled QE suggestions from https://docs.google.com/document/d/1uSHgZsXv0MR9h0dkJ2Xg__ng-n3IKdcv4m7Xpq8Jv04/edit, can you review the gdoc please and make suggestions
@xiuwang can you see if you are happy with the full flow of the docs and the new changes you suggested. THanks

File: registry/configuring-registry-operator.adoc [hunk 1]
```diff
@@ -7,31 +7,36 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 [id="image-registry-on-cloud"]
-ifndef::openshift-dedicated,openshift-rosa[]
+ifndef::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 == Image Registry on cloud platforms and OpenStack
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
+endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
+ifdef::openshift-dedicated,openshift-ros,openshift-rosa-hcp[]
```
[the remark this answers is not in the document]
Comment (fmcdonal) [reply registry/configuring-registry-operator.adoc]: thanks

**Comment by fmcdonal** [comment]
@xiuwang Can you please see the local build here:
https://file.corp.redhat.com/fmcdonal/OSDOCS-11540/registry/index.html
and make any further suggestions needed from the ones already implemented on the gdoc.

**Comment by fmcdonal** [comment]
@arendej Can you please see the local build here:
https://file.corp.redhat.com/fmcdonal/OSDOCS-11540/registry/index.html
and make any further suggestions needed from the ones already implemented on the gdoc.

File: modules/registry-common-terms.adoc [hunk 2]
```diff
@@ -9,10 +9,15 @@
 This glossary defines the common terms that are used in the registry content.
 
 container::
-Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers in data center, a public or private cloud, or your local host.
+Lightweight and executable images that consist of software and all its dependencies. Because containers virtualize the operating system, you can run containers in a data center, a public or private cloud, or your local host.
 
 Image Registry Operator::
+ifdef::openshift-rosa-hcp[]
+The Image Registry Operator runs in the `HOSTED_CLUSTERS_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry of the cluster.
```
Comment (xiuwang) [inline modules/registry-common-terms.adoc:16]: `HOSTED_CLUSTERS_NAMESPACE` need replace to `CONTROL_PLANE_NAMESPACE` to keep consistent with https://docs.openshift.com/container-platform/4.16/hosted_control_planes/hcp_high_availability/hcp-backup-restore-on-premise.html#hosted-cluster-etcd-backup-restore-on-premise_hcp-backup-restore-on-premise 

In this file, HOSTED_CLUSTERS_NAMESPACE is `clusters` namespeace

File: registry/configuring-registry-operator.adoc [hunk 3]
```diff
@@ -7,31 +7,36 @@ include::_attributes/common-attributes.adoc[]
...
-endif::openshift-dedicated,openshift-rosa[]
+endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 
-After the control plane deploys, the Operator creates a default `configs.imageregistry.operator.openshift.io` resource instance based on configuration detected in the cluster.
+After the control plane deploys in the management cluster, the Operator creates a default `configs.imageregistry.operator.openshift.io` resource instance based on configuration detected in the cluster.
 
 If insufficient information is available to define a complete `configs.imageregistry.operator.openshift.io` resource, the incomplete resource is defined and the Operator updates the resource status with information about what is missing.
 
+ifdef::openshift-rosa-hcp[]
+The Image Registry Operator runs in the `HOSTED_CLUSTERS_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry namespace of the cluster. All configuration and workload resources for the registry reside in that namespace.
```
Comment (xiuwang) [inline registry/configuring-registry-operator.adoc:33]: Same as above

File: modules/registry-common-terms.adoc (hunk 2 above)
Comment (fmcdonal) [reply modules/registry-common-terms.adoc:16]: Done thanks

**Comment by xiuwang** [comment]
LGTM

File: modules/registry-common-terms.adoc [hunk 4]
```diff
@@ -9,10 +9,15 @@
 This glossary defines the common terms that are used in the registry content.
 
 container::
-Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers in data center, a public or private cloud, or your local host.
+Lightweight and executable images that consist of software and all its dependencies. Because containers virtualize the operating system, you can run containers in a data center, a public or private cloud, or your local host.
 
 Image Registry Operator::
+ifdef::openshift-rosa-hcp[]
```
Comment (EricPonvelle) [inline modules/registry-common-terms.adoc:15]: This line and line 18 are in the wrong place. I think this section needs to be like so:
```suggestion
ifdef::openshift-rosa, openshift-rosa-hcp[]
Image Registry Operator::
ifdef::openshift-rosa-hcp[]
The Image Registry Operator runs in the `CONTROL_PLANE_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry of the cluster.
endif::openshift-rosa-hcp[]
ifdef::openshift-rosa[]
The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
endif::openshift-rosa[]
endif::openshift-rosa,openshift-rosa-hcp[]
```

**Review by EricPonvelle — COMMENTED** [review]
Small fix for the conditionals.

File: registry/configuring-registry-operator.adoc (hunk 3 above)
Comment (fmcdonal) [reply registry/configuring-registry-operator.adoc]: @EricPonvelle Do I need to add this also here as suggested below:
ifdef::openshift-rosa,openshift-rosa-hcp[]
ifdef::openshift-rosa-hcp[]
The Image Registry Operator runs in the `CONTROL_PLANE_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry namespace of the cluster. All configuration and workload resources for the registry reside in that namespace.
endif::openshift-rosa-hcp[]
ifdef::openshift-rosa[]
The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location as well. All configuration and workload resources for the registry reside in that namespace.
endif::openshift-rosa[]
endif::openshift-rosa,openshift-rosa-hcp[]

File: modules/registry-checking-the-status-of-registry-pods.adoc [hunk 5]
```diff
@@ -36,8 +36,7 @@ $ oc get pods -n openshift-image-registry
 [source,terminal]
 ----
 NAME READY STATUS RESTARTS AGE
-cluster-image-registry-operator-764bd7f846-qqtpb 1/1 Running 0 78m
-image-registry-79fb4469f6-llrln 1/1 Running 0 77m
+                                                                                                                                                    image-registry-79fb4469f6-llrln 1/1 Running 0 77m
```
Comment (xiuwang) [inline modules/registry-checking-the-status-of-registry-pods.adoc:39]: @fmcdonal This line is broken

Comment (fmcdonal) [reply modules/registry-checking-the-status-of-registry-pods.adoc]: removed spaces, thanks

**Comment by xiuwang** [comment]
LGTM

File: modules/registry-common-terms.adoc (hunk 4 above)
Comment (fmcdonal) [reply modules/registry-common-terms.adoc:16]: got it thanks

File: modules/registry-checking-the-status-of-registry-pods.adoc (hunk 5 above)
Comment (fmcdonal) [reply modules/registry-checking-the-status-of-registry-pods.adoc]: removed the spacing

**Pull request closed without merging**